xbmc/rendering/dx/DeviceResources.cpp

```diff
@@ -716,26 +716,14 @@ void DX::DeviceResources::ResizeBuffers()
 
     if (CServiceBroker::GetLogging().IsLogLevelLogged(LOGDEBUG))
     {
-      ComPtr<IDXGISwapChain4> swapChain4;
-      if (SUCCEEDED(m_swapChain.As(&swapChain4)))
+      std::vector<DXGI_COLOR_SPACE_TYPE> cs = SwapChainColorSpaces();
```

Comment:
This variable `cs` is not really necessary. You can can directly `SwapChainColorSpaces` inside for loop:

`for (const auto& colorSpace : SwapChainColorSpaces())`

Reply:
yes